api/src/common/responses.py

```diff
@@ -17,7 +17,7 @@
 )
 from common.logger import logger
 
-responses = {
+responses: dict[int | str, dict[str, Any]] = {
```

Comment:
Just a note:
looks like there is a bit of inconsistency in use of the type "dict" vs "Dict".

not sure what is recommended to use is...

Could perhaps add new issue for this.

Reply:
According to [this Stack Overflow answer](https://stackoverflow.com/a/67190449/11192976) there's no longer any difference between them.  Therefore, as far as I can see, there's no longer any need for importing `typing.List` and `typing.Dict`, as the built-in `list` and `dict` serve the same purpose. 

Feel free to correct me if I'm wrong.

Reply:
nice! Then its probably better to use the built in types